dashboard/db/migrate/20170120193705_change_id_column_type.rb

```diff
@@ -0,0 +1,9 @@
+class ChangeIdColumnType < ActiveRecord::Migration[5.0]
+  def up
+    change_column :activities, :id, :bigint
+  end
+
+  def down
+    change_column :activities, :id, :integer
```

Comment:
No idea. But since `change_column` isn't reversible by default, it seems better to have this (even if data is lost) as an option should we decide to exercise it?